cmd/kat-server/server.go

```diff
@@ -114,7 +115,80 @@ func main() {
 			Port: Port,
 		}
 		listeners = append(listeners, s)
+	case "health_check_server":
+		port := Port
+		securePort := SSLPort
+
+		HealthyStatusCode, err := strconv.Atoi(os.Getenv("HEALTHY_STATUS_CODE"))
+		if err != nil {
+			HealthyStatusCode = 200
+		}
+		UnhealthyStatusCode, err := strconv.Atoi(os.Getenv("UNHEALTHY_STATUS_CODE"))
+		if err != nil {
+			UnhealthyStatusCode = 500
+		}
 
+		for {
```

Comment:
I was just copying the setup for the main kat server (since the health checks server is pretty similar) but I removed the loop and it seems fine without it.

Reply:
OK cool :).